openwisp_radius/receivers.py

```diff
@@ -44,3 +44,11 @@ def organization_post_save(instance, **kwargs):
         rg.name = rg.name.replace(instance.__old_slug, instance.slug)
         rg.full_clean()
         rg.save()
+
+
+def radiustoken_post_save(instance, **kwargs):
+    instance.change_in_cache()
+
+
+def radiustoken_post_delete(instance, **kwargs):
+    instance.delete_cache()
```

Comment:
@nemesisdesign Okay, after reading some of your reviews, my understanding was that you suggest moving the operation of the signals inside the model itself and calling the function from the model itself, this hasn't been done anywhere else so I still have doubts that I've understood you correctly or not, please confirm! :smile:

Reply:
Look here: https://github.com/openwisp/openwisp-network-topology/pull/76/files#diff-7e33562dabdb281320b3fc9a933ccfc1R16-R37

We've been using similar patterns lately.

I would define the receiver as a class method in the AppConfig class for convenience (it's easier to read and follow the code this way), then call the model method like you're doing here.